app/src/main/res/values/strings.xml

```diff
@@ -6,26 +6,26 @@
     <string name="calendar">Calendar</string>
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
-    <string name="apache"> Beta - Apache License 2.0</string>
+    <string name="apache"> Beta — Apache 2.0</string>
     <string name="kotlin">Kotlin</string>
     <string name="java">Java</string>
-    <string name="gnu">GNU General Public License CE</string>

```

Comment:
CE?

Reply:
> CE?

Classpath Exception

Reply:
OK, but it doesn't say what version of the GPL it is.